### PR TITLE
Solution for Proposal #19974

### LINKED ISSUE #19974: [Proposal] Show username when triggering 507 Insufficient Storage
It would be really nice to get the username when the system is triggering **507 Insufficient Storage** in the logs.

### PR DESCRIPTION
**Pull request by pierrejochem**
Show username when writing into owncloud.log

Fix #19974

### CODE REVIEW

File: apps/dav/lib/connector/sabre/exceptionloggerplugin.php [hunk 1]
```diff
@@ -94,6 +94,8 @@ public function logException(\Exception $ex) {
 			$message = "HTTP/1.1 {$ex->getHTTPCode()} $message";
 		}
 
+		$user = \OC_User::getUser();
```
Comment (nickvergessen) [inline apps/dav/lib/connector/sabre/exceptionloggerplugin.php]: This gives you the current user, not the owner of the storage that has e.g. Insufficient storage

Comment (DeepDiver1975) [inline apps/dav/lib/connector/sabre/exceptionloggerplugin.php]: but this is the user who will get the error displayed - which is okay from my pov.
we might want to add the path here as well???

Comment (pierrejochem) [inline apps/dav/lib/connector/sabre/exceptionloggerplugin.php]: Okay the reason for my proposal is that we get on our system the 507 Insufficient Storage error and it is always a pain in the ass to figure out which user triggert it.

**Comment by pierrejochem** [comment]
This is MIT licensed

**Comment by nickvergessen** [comment]
Thanks, I replaced your PR with https://github.com/owncloud/core/pull/22555 so our testing CI can run.

**Pull request closed without merging**